src/components/TodoList/TodoList.tsx

```diff
@@ -1,224 +1,113 @@
-/* eslint-disable */
 import React from 'react';
+import { useAppDispatch, useAppSelector } from '../../app/hooks';
+import classNames from 'classnames';
+import { Todo } from '../../types/Todo';
+import { setCurrentTodo } from '../../features/currentTodo';
 
-export const TodoList: React.FC = () => {
-  return (
-    <>
-      <p className="notification is-warning">
-        There are no todos matching current filter criteria
-      </p>
-
-      <table className="table is-narrow is-fullwidth">
-        <thead>
-          <tr>
-            <th>#</th>
-
-            <th>
-              <span className="icon">
-                <i className="fas fa-check" />
-              </span>
-            </th>
-
-            <th>Title</th>
-            <th> </th>
-          </tr>
-        </thead>
-
-        <tbody>
-          <tr data-cy="todo">
-            <td className="is-vcentered">1</td>
-            <td className="is-vcentered"> </td>
-
-            <td className="is-vcentered is-expanded">
-              <p className="has-text-danger">delectus aut autem</p>
-            </td>
-
-            <td className="has-text-right is-vcentered">
-              <button data-cy="selectButton" className="button" type="button">
-                <span className="icon">
-                  <i className="far fa-eye" />
-                </span>
-              </button>
-            </td>
-          </tr>
-
-          <tr data-cy="todo">
-            <td className="is-vcentered">2</td>
-            <td className="is-vcentered"> </td>
-
-            <td className="is-vcentered is-expanded">
-              <p className="has-text-danger">
-                quis ut nam facilis et officia qui
-              </p>
-            </td>
+function getVisibleTodos(todos: Todo[], query: string, status: string) {
+  let copyTodos = [...todos];
```

Comment:
There is a typo in the line: `let copyTodos = [.todos];`. It should be `let copyTodos = [...todos];` to correctly create a shallow copy of the todos array using the spread operator.